api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_highly_variable_genes.py

```diff
@@ -57,6 +57,7 @@ def _highly_variable_genes_seurat_v3(
     n_top_genes: int = 1000,
     layer: str = "raw",
     span: float = 0.3,
+    max_loess_jitter: float = 1e-9,
```

Comment:
Should we allow users to specify `jitter` explicitly? If specified, we don't do the search across orders of magnitude and let the function fail with `ValueError` if there is a loess error.

I worry that the function may take a long time if the following conditions were met:
 - Loess errors are common
 - `1e-18` starting value for jitter is not enough to resolve loess

In your experimenting, what are the chances of the above conditions being met? If likely, then a `jitter` parameter would let users set a reasonably high value that is likely to work so they don't have to wait for the function to scan.

Otherwise, approving now as the current implementation looks good!

Reply:
The amount of jitter needed is going to be very dataset dependent, so I think there are edge cases for almost any default value.  I picked a conservative range of values because I thought getting the minimum jitter that "solves" was more important than performance (performance being dominated by I/O - read - time in almost all cases).

The current API allows the user to control only the max jitter.  We could expand it to allow the user to specify the entire range (and step) to try. But that still leaves the issue of what a good default is, which will be where most users start.

@pablo-gar - appreciate your thoughts.